src/lib/core/route.test.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import { expect } from 'chai'
+import { decorateRouteComponent, isRouteComponent, getRouteComponentPath } from './route'
+
+describe('isRouteComponent()', () => {
+  it('should return true for decorated component', () => {
+    class MyRoute extends React.Component { }
+
+    decorateRouteComponent('/foo', MyRoute)
+    expect(isRouteComponent(MyRoute)).to.be.true
+  })
+
+<<<<<<< HEAD
```

Comment:
Conflict issues?

Reply:
Ergh, vscode is really bad at reloading changed files... Will push a fix then rebase it out.